packages/compass-indexes/src/components/search-indexes-table/search-indexes-table.spec.tsx

```diff
@@ -85,22 +98,26 @@ describe('SearchIndexesTable Component', function () {
     });
   }
 
-  it('does not render the table if there are no indexes', function () {
-    renderIndexList({
+  it.only('renders the zero state rather than the table if there are no indexes', function () {
```

Comment:
```suggestion
  it('renders the zero state rather than the table if there are no indexes', function () {
```

Reply:
Ugh I removed it but never saved the file.